cmd/minishift/cmd/delete_test.go

```diff
@@ -71,6 +73,35 @@ func Test_clear_cache_forced(t *testing.T) {
 	assert.False(t, filehelper.Exists(state.InstanceDirs.Cache), "Expected cache dir '%s' to be deleted", state.InstanceDirs.Cache)
 }
 
+func Test_remove_entries_for_cluster(t *testing.T) {
+	kubeConfigPath := filepath.Join("..", "..", "..", "test", "testdata", "kubeconfig")
+
+	dirtyKubeConfigPath := filepath.Join("..", "..", "..", "test", "testdata", "kubeconfig_dirty")
+	cleanKubeConfigPath := filepath.Join("..", "..", "..", "test", "testdata", "kubeconfig_clean")
+
+	dirtyKubeConfig, err := clientcmd.LoadFromFile(kubeConfigPath)
+	assert.NoError(t, err, "Error loading dirty kubeconfig file")
+
+	os.Rename(kubeConfigPath, dirtyKubeConfigPath)
+	os.Rename(cleanKubeConfigPath, kubeConfigPath)
+
```

Comment:
Moving the files, to keep the `LocationOfOrigin` field same in the config structs.